source/main/modules/window-manager/index.ts

```diff
@@ -121,7 +128,16 @@ export default class WindowManager {
           callingWindow.minimize()
           break
         case 'win-close':
-          callingWindow.close()
+          if (process.platform !== 'darwin') {
```

Comment:
`win-close` event fires on any window close, including closing the preferences window. Instead, only want tray function on the main window.